manifests/hpo-operator-deployment.yaml_template

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
```

Comment:
Suggest to rename this file to `hpo-deployment.yaml_template` as this is not an operator